pkg/cloud/api/path.go

```diff
@@ -92,6 +123,15 @@ func (p Path) HasPrefix(prefix Path) bool {
 	return true
 }
 
+// isAnyIndexElement checks if path matches slice path with any index
+func isAnyIndexElement(element string) bool {
+	return element == string(pathSliceIndex)+string(anyIndex)
+}
+
+func isArrayIndex(path string) bool {
```

Comment:
* Change name to `isSliceIndex`. You should always match the existing naming convention (e.g Slice vs Array) to make it easier for people to understand the code as a maintainer.
* Change parameter name to `element`. You are not getting a `path` passed into the func.

Change docstring comment: `// isArrayIndex returns true element is a SliceIndex`